xain_fl/fl/coordinator/coordinator.py

```diff
@@ -123,7 +122,7 @@ def fit(
         return hist_co, hist_ps, hist_opt_configs, hist_metrics
 
     def fit_round(
-        self, indices: List[int], E: int
+        self, indices: List[str], E: int
```

Comment:
this is how it has been implemented on the `grpc` side, which makes sense because strings are way more flexible and could become hashes and so on. this file belongs to the local coordinator that will be remove/refactored anyway, so i wouldn't bother with it. i change it because i refactored `Controller.select_ids()`, which returns a `List[str]` and not `int`s.